install/rpm/build.sh

```diff
@@ -135,7 +147,7 @@ verify_channel() {
 }
 
 process_opts() {
-  while getopts ":o:b:c:a:h" OPTNAME
+  while getopts ":o:b:c:a:ph" OPTNAME
```

Comment:
Does this accidentally drop the `-h` option?

Reply:
It shouldn't, no. If my understanding is correct options without arguments do not have a colon after them. A quick test shows `-h` still working.

Reply:
I see, great!